src/RadioButton/RadioButton.component.tsx

```diff
@@ -56,10 +67,26 @@ export type Props = {
   theme?: any;
 };
 
+const displayStyleRules: Function = (
+  displayStyle: 'inline' | 'stacked',
+  theme: any,
+) => {
+  console.log(theme);
+  if (displayStyle === 'inline') {
+    return {
+      display: 'inline-block',
+      'padding-right': theme.radioButton.inlineSpacing,
+    };
+  }
+  return {
+    display: 'block',
+    'padding-bottom': theme.checkbox.stackedSpacing,
```

Comment:
This should be `theme.radioButton.stackedSpacing`.

Reply:
Fixed